### ISSUE TITLE
Does not build/run with non-relative paths giving errors

### ISSUE THREAD

**Issue by amritabithi** [evias/blockchain-cli#5]
I would like to run this project but cannot get it to work, I got the composer part figured out but always get this error, and the files it is looking for in /vendor/ are not present after using composer to install the dependencies.  Is there something I am missing installing?

PHP Fatal error:  require(): Failed opening required '/home/myuser/omni-recovery-tool/bootstrap/../../../autoload.php' (include_path='.:/usr/share/php') in /home/myuser/omni-recovery-tool/bootstrap/autoload.php on line 22

**Comment by evias** [comment]
Have you tried `composer install --prefer-dist` ?

Also, are you using PHP7 ?

After installing dependencies, run `php application list` to view available commands.

**Comment by amritabithi** [comment]
Thank you for the response, I tried a fresh download and used "composer install --prefer-dist" to get the dependencies and have the same error.  I may not be familiar enough with composer or something though because I feel like I am just missing something obvious.

If I trace through the script starting at application, application requires() the file init.php, which then requires() the file autoload.php, and then inside of autoload.php it tries to include another autoload.php three directories above itself as the last command before it crashes, and so I wonder if the enivornment is different.

( Also I checked and I have PHP 7.4.3 (cli) and am using Ubuntu 20 )

**Comment by evias** [comment]
weirdly, maybe you would need `composer dump-autoload` after the install ; Seems like its not automatically being run.

**Comment by amritabithi** [comment]
I tried the 'composer dump-autoload' command and while I still get the same errors as before there is a new error " Uncaught Error: Class 'LaravelZero\Framework\Application' not found".  I also tried upgrading to composer version 2 but none of the installation commands given by the composer creators work or upgrade anything from version 1 which is all I can find available, it just says package not found etc.

Anyway I decided to use another library and I can use the logic and things from this as a guideline so that is useful, I can never get the composer projects to run for some reason.  It is almost like the project was built in a directory that is a symbolic link because the paths as they are in the include files traverse up and out of the project folder.
